mobile/library/java/org/chromium/net/CronvoyProvider.java

```diff
@@ -191,40 +165,4 @@ private static void logReflectiveOperationException(String className, boolean lo
       }
     }
   }
-
-  /**
-   * Attempts to add a provider specified in the app resource file to a set.
-   *
-   * @param providers the set of providers to add the new provider to.
-   * @return {@code true} if the provider was added to the set; {@code false} if the app resources
-   *     do not include the string with {@link #RES_KEY_CRONET_IMPL_CLASS} key.
-   * @throws RuntimeException if the provider cannot be found or instantiated.
-   */
-  private static boolean addCronetProviderFromResourceFile(Context context,
```

Comment:
Is this removal also part of the same change?

Reply:
This removal is actually landed in https://github.com/envoyproxy/envoy/pull/27039 already.
The reason for this change is that the original Cronet is able to provide a number of Cronet impls, including native (dev release), GMScore(gmscore release), and java impl(a fallback impl using java networking lib).
It also supports loading custom impl from a resource file.
For Cronvoy, we don't need any of these.
In fact, my Cronvoy embedder app has code that specifically checks we are using the native impl but not others. Hence I think we should remove loading unnecessary resources.